### ISSUE TITLE
The Import Cannot Be Resolved, IntelliJ and VS Code

### ISSUE THREAD

**Issue by akcorp2003** [redhat-developer/vscode-java#366]
I'm getting an issue `The import com.[company name].taskservice cannot be resolved` on a project. I was getting this issue first and then I ran a `mvn idea:idea` and it resolved the dependencies. IntelliJ is showing up as resolved but on the VS Code plugin, the imports still cannot be resolved. 

It looks like the `mvn idea:idea` generated an additional folder named `target`  at the same directory level as the `pom.xml` and all the additional dependencies get stored there.

Is this part of the IntelliJ import issue  #191 ?

##### Environment
- Operating System: Mac OS X 10.12.6
- JDK version: 1.8.0_131
- Visual Studio Code version: 1.18.0
- Java extension version: 0.14.0

**Comment by fbricon** [comment]
Can you check how the project imports in Eclipse? Can you create a sample project reproducing the issue?

**Comment by akcorp2003** [comment]
I imported the same project into Eclipse and I'm getting the same unresolvable errors. Unfortunately, I'm not familiar with the project so I don't know how to structure a sample one to reproduce this issue. My company is using an internal maven repo.

**Comment by gorkem** [comment]
@akcorp2003 is the unresolved `com.[company name].taskservice ` available only on the internal maven repo?

**Comment by akcorp2003** [comment]
@gorkem Yes. It's an internal maven repo.

**Comment by fbricon** [comment]
are you using a proxy to connect to your repo? if so, what kind?

**Comment by akcorp2003** [comment]
Yes. We do have a proxy. But unfortunately, I don't know what kind of proxy it is. We use a .pac file.

**Comment by ImATechnophile** [comment]
Just clean the workspace directory.

You can clean the workspace directory in the following way:

Open the command palette (F1)
select Java: Clean the Java language server workspace
click Restart and delete

**Comment by vovasoft** [comment]
> Just clean the workspace directory.
> 
> You can clean the workspace directory in the following way:
> 
> Open the command palette (F1)
> select Java: Clean the Java language server workspace
> click Restart and delete
This way is useful, Thx

**Comment by akcorp2003** [comment]
Closing issue as I don't need it open anymore.

**Comment by RenanZX** [comment]
I did it, but it's not working

**Comment by mkhaled7** [comment]
> I did it, but it's not working

Were you able to fix it?

**Comment by RenanZX** [comment]
> Were you able to fix it?

No, i needed to create a new project in vscode, and copy paste my files from the project that i created before.

**Comment by mkhaled7** [comment]
> > Were you able to fix it?
> 
> No, i needed to create a new project in vscode, and copy paste my files from the project that i created before.

I tried the same but got the same errors.

**Comment by mkhaled7** [comment]
> > I tried the same but got the same errors.
> 
> Create a new clean project, and see if you can access the hello world page normally, if you can, copy and paste just folder java and pages from your webapp folder. If you get a error message again, create a new project and repeat the process, if you receive the same error no metter how many new projects you create, so try to reinstall your VScode-java again.

Thank you so much for such quick response! You are very kind.

**Comment by RenanZX** [comment]
> Thank you so much for such quick response! You are very kind.

You're welcome.

**Comment by zkxjzmswkwl** [comment]
This is still an issue.

There very much needs to be better support of Maven projects.

**Comment by Tobyha01** [comment]
So I get the issue and other issues when trying to import from the `jakarta` library, however if import the same things from `javax`, the issues seem to go away, I have tried running the java clean several times by doing control shift p, and searching java clean, this does not work. Another fix I have seen is finding the code file in the app data on your computer and then deleting the subfolder labelled local storage, this has also not worked for me, I have just tried running the `mvn idea:idea` as suggested, I don't know if there is anything extra to be included in the terminal, as when I literally just typed that I kept getting command not found, I tried both in command prompt and bash terminal (I'm on a windows machine). It may also be worth noting that this is a fresh project I started on this machine, I am not trying to open an existing project. I went to the spring initialiser page and generated a project specifically for a task and started it from scratch as it were. I know if I was to use eclipse or IntelliJ these issues probably wouldn't happen as much, although some people still seem to have issues despite using those ide's. If anyone else has further suggestions on what I could try, would be most appreciated.
